Formula/c/cdxgen.rb

```diff
@@ -14,6 +14,7 @@ class Cdxgen < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "307568fcf2426432b90623712c58a7f3cb5281b997722cc6aad828cb09927428"
   end
 
+  depends_on "ruby" => :test
```

Comment:
During the test, it doesn't found `libruby.so.3.4` and so that is why I add this here.